platform-sdk/swirlds-logging/src/main/resources/META-INF/services/com.swirlds.logging.api.extensions.handler.LogHandlerFactory

```diff
@@ -0,0 +1 @@
+com.swirlds.logging.console.ConsoleHandlerFactory
```

Comment:
We now can use Google Autoservice to generate that file. 
Example for config-impl:

annotation processor def in gradle file: https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-config-impl/build.gradle.kts
annotation usage: https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigurationBuilderFactoryImpl.java